.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: release
```

Comment:
NIT: Add license header?

# Copyright 2020 Vectorized, Inc.
#
# Use of this software is governed by the Business Source License
# included in the file licenses/BSL.md
#
# As of the Change Date specified in that file, in accordance with
# the Business Source License, use of this software will be governed
# by the Apache License, Version 2.0

Reply:
Thanks, since this is not redpanda itself it will just be the apache license.